README.md

```diff
@@ -1,172 +1,118 @@
-| | | eu-west-1: ami-0d929d41d1f580dd5 |
-| | | eu-west-2: ami-0cbb336c0ee02142b |
-| | | eu-west-3: ami-02daf41cab4e66639 |
-| | | sa-east-1: ami-08434c76fb33b7dea |
-| | | us-east-1: ami-06d15e7a5aa71bd81 |
-| | | us-east-2: ami-002c5b8ed125b204d |
-| | | us-west-1: ami-0f14c305334254b09 |
-| | | us-west-2: ami-0cbf198692b35cc70 |
-| | Ubuntu Bionic | ap-northeast-1: ami-01b9b95604fddade1 |
-| | | ap-northeast-2: ami-0429d18e76a0b3705 |
-| | | ap-south-1: ami-09930fca077b07e18 |
-| | | ap-southeast-1: ami-0b2e5665eda719758 |
-| | | ap-southeast-2: ami-01c2df5ce9bc2f573 |
-| | | ca-central-1: ami-0f5fbb71a0c65e000 |
-| | | eu-central-1: ami-0fad7824ed21125b1 |
-| | | eu-west-1: ami-0da760e590e7de0e8 |
-| | | eu-west-2: ami-04137690b33cb5a8e |
-| | | eu-west-3: ami-028272d8c8e9ff369 |
-| | | sa-east-1: ami-041a08e5511d25535 |
-| | | us-east-1: ami-0de61b6929e8f091c |
-| | | us-east-2: ami-0a2463ac1e1f46b95 |
-| | | us-west-1: ami-05dc1567db5bd869a |
-| | | us-west-2: ami-0f33a1d90f189e0a1 |
-
-## Community, discussion, contribution, and support
-
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
-
-You can reach the maintainers of this project at:
-
-- [#cluster-api on Kubernetes Slack](http://slack.k8s.io/messages/cluster-api)
-- [SIG-Cluster-Lifecycle Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
-
-### Code of conduct
-
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png" width="100">
+<img style="float: right;" src="https://d0.awsstatic.com/logos/powered-by-aws.png" alt="Powered by AWS Cloud Computing">
 
+------
+Kubernetes-native declarative infrastructure for AWS.
+
+
+
+What is the Cluster API Provider AWS?
+------------------------------------
+The [Cluster API](https://github.com/kubernetes-sigs/cluster-api) brings
+declarative, Kubernetes-style APIs to cluster creation, configuration and
+management.
+
+The API itself is shared across multiple cloud providers allowing for true AWS
+hybrid deployments of Kubernetes. It is built atop the lessons learned from
+previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
+[kubicorn](http://kubicorn.io/).
+
+Launching a Kubernetes cluster on AWS
+-------------------------------------
+
+Check out [our getting started guide](docs/getting-started.md) for launching a
+cluster on AWS.
+
+Features
+--------
+
+* Native Kubernetes manifests and API
+* Manages the bootstrapping of VPCs, gateways,
+  security groups and instances.
+* Choice of Linux distribution between Amazon Linux 2,
+  CentOS 7 and Ubuntu 18.04
+* Deploys Kubernetes control planes with support for nodes
+  in public or private subnets.
+* Doesn't use SSH for bootstrapping nodes.
+* Installs only the minimal components to bootstrap a
+  control plane and workers.
```

Comment:
Technically, we require pre-baked images that contain the minimal components needed. It might be good to call out that we do provide the tooling to create those images, though.